static-site/components/list-resources/group-and-resource-links.tsx

```diff
@@ -141,6 +143,17 @@ export function IndividualResourceLink({
         {hovered || topOfColumn
           ? resource.displayName.hovered
           : resource.displayName.default}
+        {resource.resourceType === 'narrative' && (
```

Comment:
Adding the icon at the end can cause it to be cut off for long narrative names. This is specifically on the `/groups` page, where you can only see the icon when you expand to show all resources 

<img width="998" height="147" alt="Screenshot 2025-12-04 at 2 58 12 PM" src="https://github.com/user-attachments/assets/a27616a3-7b21-40fb-8bab-6f9f0d653ad1" />

I'd suggest to add the icon to the beginning or keep `/narratives/` in the name